app/views/proprietor/accounts/_shared_search_toggle.html.erb

```diff
@@ -0,0 +1,71 @@
+
+<!-- Originally from https://github.com/samvera/hyrax/blob/main/app/views/hyrax/admin/features/index.html.erb -->
+<% feature_set = all_features  %>
+<% grouped_features = feature_for_display_in_proprietor_account_ui %>
+
+<div class="flip row">
+  <div class="col-md-12">
+    <div class="panel">
+      <div class="panel-body">
+        <div class="table-responsive">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th></th>
+                <th class="name"><%= t('.feature') %></th>
+                <th class="description"><%= t('.description') %></th>
+                <th class="action"><%= t('.action') %></th>
+              </tr>
+            </thead>
+            <tbody>
+            <%# grouped_features&.each do |group, features| -%>
+            <% grouped_features&.each do |feature| -%>
+
+              <% if feature_set.grouped? -%>
+                <tr class="group">
```

Comment:
I would extract the `tr` into a partial to make the file smaller and easier to read

Reply:
I think we should let this one stay as it is , because I copied this file from hyrax and I want it to stay close to the original  hyrax file,  so in future, if they make a change it will be easier to see the difference and update this file.